src/bosh-google-cpi/google/config/config.go

```diff
@@ -4,13 +4,27 @@ import (
 	bosherr "github.com/cloudfoundry/bosh-utils/errors"
 )
 
+var CpiRelease string
+
 type Config struct {
 	Project               string `json:"project"`
+	UserAgentPrefix       string `json:"user_agent_prefix"`
 	JSONKey               string `json:"json_key"`
 	DefaultRootDiskSizeGb int    `json:"default_root_disk_size_gb"`
 	DefaultRootDiskType   string `json:"default_root_disk_type"`
 }
 
+func (c Config) GetUserAgent() string {
+	boshCpiUserAgent := "bosh-google-cpi/" + CpiRelease
+	if CpiRelease == "" {
```

Comment:
real nit here but for readability:
- `userAgent` will do
- don't reassign `userAgent`. Set `CpiRelease` (or temp value) and always set `userAgent` to `"bosh-google-cpi/" + release`